components/tabs/BUILD.gn

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+assert(!is_android)
+
+static_library("public") {
```

Comment:
Having static_lib only with headers seems cause failure on Windows.
It warns that "no input files, not writing output file pass /llvmempty to write empty.lib file...". I guess :public and :tabs should be merged.